talktorials/11_online_apis/11b_online_docking.ipynb

```diff
@@ -0,0 +1,910 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Talktorial 11 (part B)\n",
+    "\n",
+    "# CADD web services that can be used via a Python API\n",
+    "\n",
+    "__Developed at AG Volkamer, Charité__\n",
+    "\n",
+    "Dr. Jaime Rodríguez-Guerra\n",
+    "\n",
+    "## Aim of this talktorial\n",
+    "\n",
+    "> This is part B of the \"Online webservices\" talktorial:\n",
+    ">\n",
+    "> - 11a. Querying KLIFS & PubChem for potential kinase inhibitors\n",
+    "> - __11b. Docking the candidates against the target obtained in 11a__\n",
+    "> - 11c. Assessing the results and comparing against known data\n",
+    "\n",
+    "\n",
+    "After obtaining input structures we will use molecular docking software to find good protein-ligand poses.\n",
+    "\n",
+    "## Learning goals\n",
+    "\n",
+    "### Theory\n",
+    "\n",
+    "- Molecular docking basics\n",
+    "- Available software\n",
+    "- Known limitations\n",
+    "\n",
+    "### Practical\n",
+    "\n",
+    "- Prepare the structures\n",
+    "- Guess the binding site\n",
+    "- Run the calculation\n",
+    "- Save the results\n",
+    "\n",
+    "### Discussion\n",
+    "\n",
+    "See below.\n",
+    "\n",
+    "### Quiz\n",
+    "\n",
+    "See below.\n",
+    "\n",
```

Comment:
Discussion and Quiz sections not necessary here (did not have them in talktorials 1-10).